lib/flight_job/outputs/info_job.rb

```diff
@@ -28,107 +28,99 @@
+        register(header: 'Started at', &:actual_start_time)
       else
-        Outputs.format_time(job.estimated_end_time, false)
+        register(header: 'Estimated Start', &:estimated_start_time)
       end
-    end
 
-    path_header = ->(job, verbose:) do
-      if job.stdout_path == job.stderr_path && !verbose
+      if job.actual_end_time || verbose?
+        register(header: 'Ended at', &:actual_end_time)
+      else
+        register(header: 'Estimated Finish', &:estimated_end_time)
+      end
+
+      stdout_header = if job.stdout_path == job.stderr_path && !verbose?
         'Output Path'
       else
         'Stdout Path'
       end
-    end
-    register_attribute(header: path_header) { |j| j.stdout_path }
-    register_attribute(header: 'Stderr Path', verbose: true) { |j| j.stderr_path }
-
-    register_attribute(section: :submit, header: 'Submit Stdout') do |job|
-      job.submit_stdout
-    end
-    register_attribute(section: :submit, header: 'Submit Stderr') do |job|
-      job.submit_stderr
-    end
+      register(header: stdout_header) { job.stdout_path }
+      if verbose?
+        register(header: 'Stderr Path') { job.stderr_path }
+      end
 
-    # NOTE: The following appear after the submit attributes in the non-interactive output. This
-    # maintains the column order and backwards compatibility.
-    #
-    # The submit columns will always be sorted to the bottom in the interactive outputs.
-    #
-    # Consider reordering on the next major version bump.
-    register_attribute(header: 'Results Dir') { |j| j.results_dir }
-    register_attribute(verbose: true, header: 'Estimated start') do |job|
-      Outputs.format_time(job.estimated_start_time, false)
-    end
-    register_attribute(verbose: true, header: 'Estimated end') do |job|
-      Outputs.format_time(job.estimated_end_time, false)
-    end
+      # Display the stdout/stderr callables in non-interactive
+      # They are hard rendered into the interactive template
+      unless interactive?
+        register(header: 'Submit Stdout') { job.submit_stdout }
+        register(header: 'Submit Stderr') { job.submit_stderr }
+      end
 
-    register_attribute(header: "Desktop ID:", &:desktop_id)
+      # NOTE: The following appear after the submit attributes in the non-interactive output. This
+      # maintains the column order and backwards compatibility.
+      #
+      # The submit columns will always be sorted to the bottom in the interactive outputs.
+      #
+      # Consider reordering on the next major version bump.
+      register(header: 'Results Dir') { job.results_dir }
+      if verbose?
+        register(header: 'Estimated start', &:estimated_start_time)
+        register(header: 'Estimated end', &:estimated_end_time)
+      end
 
-    def self.build_output(**opts)
-      submit = opts.delete(:submit)
-      super(template: TEMPLATE, context: { submit: submit }, **opts)
+      register(header: "Desktop ID:", &:desktop_id)
```

Comment:
The `:` needs removing.

Reply:
4a6fc11c3917734bedb2b2088ae338a39fbdfa68